src/main/java/to/us/resume_builder/request/PostRequest.java

```diff
@@ -35,7 +35,7 @@ public PostRequest(String specificPath, String rssName) {
     }
 
     @Override
-    protected URI getURI(String... arguments) {
+    public URI getURI(String... arguments) {
```

Comment:
Why is this public? It's supposed to be protected (it's a primitive method for a template method).

That, and we'll need to update the version on this repo with the updated version from the client repo where I fixed the naming convention.